cmd/controlplane/controller.go

```diff
@@ -183,6 +186,7 @@ func (o *controllerOptions) setupKargoManager(
 			Metrics: server.Options{
 				BindAddress: "0",
 			},
+			PprofBindAddress: o.PprofBindAddress,
```

Comment:
I see no reason this _wouldn't_ be the case, but I just want to verify...

This is _just_ about the address/port for the server, right? Being associated with this manager doesn't limit visibility into things this manager isn't responsible for, right? 🤞 I ask because this controller uses _two_ managers.

Reply:
No, it's detached from the manager in the broader sense.

It gives the simple task to the manager to activate the `pprof` code path which makes Go do its magic (for the runtime as a whole), and then register an endpoint for it.

Reply:
I assumed so. Thank you for confirming! 👍